WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -15805,6 +15828,7 @@ e_armenia = {
 				OR = {
 					ai = no
 					religion_group = christian
+					culture = armenian
```

Comment:
No de jure structure which is present in the mod represents any "default"in any way. That is purely and simply wrong, and you have completely misunderstood the concept of de jure, which I explain above.

In vanilla:
> Each duchy, kingdom and empire has a lawfully right land area in the game.

As you see, your description is off the track here.

> There is a major reason why the Vanilla rendition does not limit them.

Incorrect. See the list of vanilla limitations here:
http://www.ckiiwiki.com/Empires